image_width: 0
image_height: 0
camera_name: front_cam
camera_matrix:
  rows: 3
  cols: 3
  data: [455.6140768, 0.0, 383.79553680000004, 0.0, 228.4593332, 120.6182132, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.035117, -0.09088099999999999, 0.00013, -0.000116, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  # data: [444.3081544, 0.0, 383.61721520000003, 0.0, 0.0, 229.14348140000004, 120.67244860000001, 0.0, 0.0, 0.0, 1.0, 0.0]
  data: [222.154772, 0.0, 383.61721520000003, 0.0, 0.0, 229.14348140000004, 120.67244860000001, 0.0, 0.0, 0.0, 1.0, 0.0]
roi:
  x_offset: 0
  y_offset: 0
  height: 0
  width: 0
  do_rectify: False
  